doc/Changelog.md

```diff
@@ -3,6 +3,8 @@ Changelog {#changelog}
 
 # Release 1.3 (git master)
 
+* [127](https://github.com/BlueBrain/Tide/pull/127):
+  Pixel streams decode faster by directly rendering YUV frames on GPU.
```

Comment:
Ouch, I just tired to measure locally and it's quite disappointing. I can't see any difference for a regular YUV444 stream. It seems that unlike ffmpeg, turbo-jpeg has very optimized code for decoding directly to rgb, so there is little gain in CPU usage. The lower CPU->GPU bandwidth is an advantage when using YUV420 instead of YUV444, though. When streaming my desktop the frame rate increases from 20 -> ~23-30 fps. We should do more testing later with high-resolution streams on the wall, once we have a recent enough libturbojpeg to use the feature...